### DISCUSSION TITLE
Add 0.5 second power off delay.

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#5130] [Ideas]
Hi all! (Again)

When you select a power off delay in EdgeTX, you have the options for 1s and up. However, an option would be really nice for 0.5 seconds (nice balance between accidentally turning it off and having to wait for a whole second before shutdown). 

_One unrelated thing: when I have a feature request do I put it in Issues-Feature Request or here in Discussions? What changes between both options?_

**Comment by pfeerick** [comment]
If it is an actual feature request rather than a "maybe this would be nice,
does anyone else think that" it would be a feature request. One obviously
adds to the number of issues (bugs, enhancement requests), the other
doesn't.

**Reply by inventor7777** [reply]
Makes sense, thanks for explaining! Should I open this as a separate feature request then, or just leave it as discussion?

**Comment by pfeerick** [comment]
It's probably worth going to a feature request, but I think there is an
option to convert it directly to an issue.

IIRC you are currently working directly with the offset being added to the
built in 2 second delay, so there is no need for extra strings conditions
etc atm. i.e. basically either adding -2, -1, 0 to delay, and then
performing it.

**Comment by inventor7777** [comment]
> IIRC you are currently working directly with the offset being added to the
built in 2 second delay, so there is no need for extra strings conditions
etc atm. i.e. basically either adding -2, -1, 0 to delay, and then
performing it.

What do you mean? The only options I see are 0s, 1s, and above for power off delay (0s is just click and it turns off, as opposed to the power on delay which has is mostly hardware related unoverrideable _(is that even a word!?)_ 1s or so delay, even when set to 0s)

**Reply by pfeerick** [reply]
And that is the code? :lol:

**Reply by inventor7777** [reply]
Sorry, I thought you were talking about the option on the radio. I’m not super familiar with the EdgeTX code 🙂
